cmd/ndm_daemonset/controller/controller.go

```diff
@@ -171,17 +171,16 @@ func (c *Controller) setKubeClient(cfg *rest.Config) error {
 	return nil
 }
 
-/*
 // setClientset set Clientset field in Controller struct
-// if it gets Clientiset from cfg else it returns error
-func (c *Controller) setClientSet(cfg *rest.Config) error {
-	crdClient, err := clientset.NewForConfig(cfg)
+// if it gets Client from config else it returns error
+func (c *Controller) setClientSet() error {
```

Comment:
can we rename this to `newClientSet()`

I guess, there is another method called `getClientSet()` that returns the cached copy